qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -1292,7 +1293,8 @@ def figure_names(self) -> List[str]:
 
     @property
     def share_level(self) -> str:
-        """Return the share level fo this experiment.
+        """Return the share level fo this experiment,
+           to self and all its descendants.
```

Comment:
I don't think "to self and all its descendants" is needed here because it conflicts with the `share_level` docstring below, and also because the recursive setting is in the `ExperimentData` class, not here.

Reply:
In addition it's incorrect because it's a property and not a setter

Reply:
Done